debian/changelog

```diff
@@ -3,6 +3,7 @@ cloud-init (22.1-14-g2e17a0d6-0ubuntu1~18.04.4) UNRELEASED; urgency=medium
   * d/control:
     - add python3-responses for unittests
     - add python3-pytest-mock for unittests
+    - add Suggests: openssh-server
     - add Suggests: ssh-import-id
```

Comment:
Since we haven't officially released this changelog anywhere (due to UNRELEASED state) let's consolidate these 4 lines into two:
```suggestion
    - Build-Depends: add python3-responses and python3-pytest-mock for unittests
    - Suggests: add openssh-server and ssh-import-id
```